### PR TITLE
Set up travis.

### PR DESCRIPTION
**Pull request by chowchow316**
Borrowed from service-control-client.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+install:
+  - export JAVA_HOME=/usr/lib/jvm/java-8-oracle
+  - export CC=gcc-4.8
+  - export CXX=g++-4.8
+  - wget 'https://github.com/bazelbuild/bazel/releases/download/0.4.1/bazel-0.4.1-installer-linux-x86_64.sh' -O ./bazel-installer.sh
+  - chmod +x ./bazel-installer.sh
+  - ./bazel-installer.sh  --user
+
+script:
+  - script/setup
```
Comment (qiwzhang) [inline .travis.yml]: why setup is needed?  If so, it should be in "install" or "pre_install"

Comment (chowchow316) [inline .travis.yml]: Still testing it, as I couldn't enable travis for this repo, asked @mandarjog for help. Can you enable it?

**Comment by chowchow316** [comment]
Travis has been set up.

**Comment by chowchow316** [comment]
There is no bazel WORKSPACE in this PR yet, just added check-style and make travis pass.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+install:
+  - export JAVA_HOME=/usr/lib/jvm/java-8-oracle
+  - export CC=gcc-4.8
+  - export CXX=g++-4.8
+  - wget 'https://github.com/bazelbuild/bazel/releases/download/0.4.3/bazel-0.4.3-installer-linux-x86_64.sh' -O ./bazel-installer.sh
+  - chmod +x ./bazel-installer.sh
+  - ./bazel-installer.sh  --user
+
+script:
+  - script/check-style
```
Comment (qiwzhang) [inline .travis.yml]: please add "new line" here

Comment (chowchow316) [inline .travis.yml]: Done.

**Pull request merged**